modules/light-clients/08-wasm/keeper/msg_server.go

```diff
@@ -48,6 +49,11 @@ func (k Keeper) RemoveCodeHash(goCtx context.Context, msg *types.MsgRemoveCodeHa
 		return nil, errorsmod.Wrap(err, "failed to remove code hash")
 	}
 
+	// unpin the code from the vm in-memory cache
+	if err := k.wasmVM.Unpin(msg.CodeHash); err != nil {
+		return nil, errorsmod.Wrapf(err, "failed to unpin contract with code hash (%s) from vm cache", hex.EncodeToString(msg.CodeHash))
```

Comment:
Should we return error or just continue so that we don't block removing a code hash if, for whatever reason, unpin fails?